operator/pkg/resources/lb_service.go

```diff
@@ -143,3 +143,73 @@ func (r *LoadBalancerServiceResource) getAnnotation() map[string]string {
 	}
 	return ext.External.Bootstrap.Annotations
 }
+
+// RenderLoadBalancerService renders a LoadBalancer service
+func RenderLoadBalancerService(
+	_ context.Context,
+	cluster *vectorizedv1alpha1.Cluster,
+) (k8sclient.Object, error) {
+	var bootstrapPorts []NamedServicePort
+
+	for _, listener := range cluster.KafkaAPIExternalListeners() {
+		if listener.External.Bootstrap != nil {
+			portName := listener.Name
+			if portName == "" {
+				portName = "kafka-external-bootstrap"
+			} else {
+				portName = portName + "-bootstrap"
+			}
+
+			bootstrapPorts = append(bootstrapPorts, NamedServicePort{
+				Name:       portName,
+				Port:       listener.External.Bootstrap.Port,
+				TargetPort: listener.Port,
+			})
+		}
+	}
+
+	if len(bootstrapPorts) == 0 {
```

Comment:
This is a funky case. Can you explain when/why this would happen? It's also worth mentioning such a case in the doc of this function so that callers know they need to handle it.